pkg/docker/docker.go

```diff
@@ -199,7 +207,7 @@ func (rco RunContainerOptions) asDockerConfig() dockercontainer.Config {
 		Image:        getImageName(rco.Image),
 		User:         rco.User,
 		Env:          rco.Env,
-		Entrypoint:   dockerstrslice.StrSlice(rco.Entrypoint),
+		Entrypoint:   rco.Entrypoint,
```

Comment:
dockerstrslice.StrSlice wraps []string just adding UnmarshalJSON() to unmarshal a string or array of strings in json to a []string.  In all other respects dockerstrslice.StrSlice is a []string.  Since we are not unmarshalling here nor would it make sense to, I removed the reference to dockerstrslice.StrSlice to improve readability and save the import.